src/app/components/shopping-cart/product-list/product-list-item/product-list-item.component.html

```diff
@@ -0,0 +1,20 @@
+<div class="card mb-4 shadow-sm">
+    <h1>ABC</h1>
+    <img src="../../../../../assets/baby/baby-dove.jpg" class="bd-placeholder-img card-img-top" width="100%">
+    <div class="card-body">
+      <p class="card-text">This is a wider card with supporting text below as a natural lead-in to additional content. This content is a little bit longer.</p>
+      <div class="item-info">  
+        <small>MRP Rs. {{ 100 }}</small>
+            <button type="button" class="item-btn">Buy Now</button> 
+    </div>
+    </div>
+  </div>
+
+  <!--      <div class="d-flex justify-content-between align-items-center">
+            <div class="btn-group">
+            <button type="button" class="btn btn-sm btn-outline-secondary">View</button>
+            <button type="button" class="btn btn-sm btn-outline-secondary">Edit</button>
+            </div>
+            <small class="text-muted">9 mins</small>
+      </div>
+    -->
```

Comment:
seems like an incomplete template for now. Keep working

Reply:
I have worked on this and will commit the code shortly.